hydroflow/src/util/tcp.rs

```diff
@@ -83,60 +83,63 @@ pub async fn bind_tcp<T: 'static, Codec: 'static + Clone + Decoder + Encoder<T>>
 
     let bound_endpoint = listener.local_addr()?;
 
-    let (tx_egress, mut rx_egress) = unsync_channel(None);
-    let (tx_ingress, rx_ingress) = unsync_channel(None);
-
-    let clients = Rc::new(RefCell::new(HashMap::new()));
-
-    spawn_local({
-        let clients = clients.clone();
-
-        async move {
-            while let Some((payload, addr)) = rx_egress.next().await {
-                let client = clients.borrow_mut().remove(&addr);
-
-                if let Some(mut sender) = client {
-                    let _ = SinkExt::send(&mut sender, payload).await;
-                    clients.borrow_mut().insert(addr, sender);
-                }
-            }
-        }
-    });
+    let (send_egress, mut recv_egress) = unsync_channel::<(T, SocketAddr)>(None);
```

Comment:
Curious, what was wrong with `tx` and `rx`?

Reply:
I just prefer the more explicit naming, consistent with how rust names channels